docs/content/dagster-cloud/deployment/serverless.mdx

```diff
@@ -37,14 +37,47 @@ Enterprise customers may request a quota increase by [contacting Sales](mailto:s
 
 ## Getting started with Serverless
 
```

Comment:
In other folks' guides, I've had them add a list of subsections with links in cases like this. It lets the user see what's in the section while remaining inline, and also allows them to easily jump to the next section.

```suggestion

- [With GitHub](#with-github)
- [Without GitHub (GitLab, BitBucket, or local development](#without-github-gitlab-bitbucket-or-local-development)
- [Adding secrets](#adding-secrets)

```